sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -487,15 +484,29 @@ export class StudentService extends RecordDataModelService<Student> {
       .innerJoin("student.sinValidation", "sinValidation")
       .where("user.isActive = true");
     if (institutionId) {
-      searchQuery.andWhere(
-        new Brackets((qb) => {
-          qb.where("offeringInstitution.id = :institutionId", {
-            institutionId,
-          }).orWhere("pirInstitution.id = :institutionId", { institutionId });
-        }),
-      );
+      searchQuery
+        .andWhere("institution.id = :institutionId", { institutionId })
+        .andWhere(
```

Comment:
If my assumption is right in https://github.com/bcgov/SIMS/pull/1887/files#r1167302495 we may not need these conditions.